lib/server.js

```diff
@@ -48,9 +53,16 @@ Server.prototype.listen = function listen(port, address, callback) {
                 address = '0.0.0.0';
         }
 
+        let fam = ''
```

Comment:
Node.js's `"net"` module (since v0.3.0) has this ability built in, no need for the regexes:

```js
const net = require("net");
const ipversion = net.isIP(address);
if (ipversion === 0) throw new TypeError("address is neither valid v4 or v6");
const fam = ipversion === 4 ? "udp4" : "upd6";
```

https://nodejs.org/api/net.html#net_net_isip_input